### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by stupoid** [trinodb/trino-python-client#402] [Q&A]
Hi, I'm interested in using this library with this fix merged in d6047a8.

I see that it is already merged into master but I can't seem to find any indication of when it will be released.
Is there any schedule or trigger for releases that I can look forward to?

Thanks in advance.

**Comment by hashhar** [comment]
We plan to do a release in the beginning of September.

In meantime you can install from `pip` using `pip install git+https://trinodb/trino-python-client@d6047a80c1a772cb8240f4a81df810b0d48af4df`.
